src/components/ContentPreview.tsx

```diff
@@ -208,6 +202,11 @@ const ContentPreview: FC<ContentPreviewProps> = ({ resource, isPreview = false }
               )}
             </StyledBlockWrapper>
           )}
+          {presentationMode === SupportedFileTypes.Text && (
+            <StyledPaper elevation={2}>
+              <Typography>{contentText}</Typography>
```

Comment:
Akkurat for text kan vi vel ha med en cypress-test ?

Reply:
Fikset, og fikk lagt til interceptor på getRecourceDefaultContent som manglet.

Reply:
nydelig